src/kbmod_wf/task_impls/reproject_wu.py

```diff
@@ -12,3 +23,289 @@ def reproject_wu(input_wu=None, reprojected_wu=None, logger=None):
         f.write(f"Logged: {value} - {time.time()}\n")
 
     return reprojected_wu
+
+
+def reproject_wu(
+    original_wu_filepath: str = None,
+    uri_filepath: str = None,
+    reprojected_wu_filepath: str = None,
+    runtime_config: dict = {},
+    logger: Logger = None,
+):
```

Comment:
A quick docstring here would probably also be helpful

Reply:
Done